build.rs

```diff
@@ -6,16 +6,22 @@ use std::path::{Path, PathBuf};
 
 #[cfg(feature = "trusted-service-provider")]
 fn generate_ts_bindings(ts_include_dir: String) -> Result<()> {
-    let header = ts_include_dir.clone() + "/service/locator/interface/service_locator.h";
+    let header = ts_include_dir.clone() + "/components/service/locator/interface/service_locator.h";
+    let encoding_header = ts_include_dir.clone() + "/protocols/rpc/common/packed-c/encoding.h";
```

Comment:
There is `packed-c` in the path, what is this file for? Aren't we using protobuf?

Reply:
We are, but that's the correct file nonetheless - probably a bug. I did notify the TS team.